# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libao/hotfix-nanosleep.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libao-1.2.2/src/plugins/pulse/ao_pulse.c.vanilla	2024-07-27 13:30:07.541000000 +0200
+++ libao-1.2.2/src/plugins/pulse/ao_pulse.c	2024-07-27 13:30:51.002000000 +0200
@@ -31,6 +31,7 @@
 #include <string.h>
 #include <signal.h>
 #include <limits.h>
+#include <time.h>
 
 #include <pulse/pulseaudio.h>
 #include <pulse/simple.h>
